pkg/inputs/mysqlbatch/mysql_table_scanner.go

```diff
@@ -431,6 +437,9 @@ func (tableScanner *TableScanner) AfterMsgCommit(msg *core.Msg) error {
 		return errors.Trace(err)
 	}
 
+	if err := IncrementScanCount(tableScanner.positionCache, *msg.InputStreamKey); err != nil {
```

Comment:
the scan count should be saved along with current position. why not change it in the table position `p` and save in `PutCurrentPos`?

Reply:
@Ryan-Git changed seiralization format for mysqlbatch.

Not intended to put the scan count into `Current` position.

`Current`, `Max`, `Min` use the same struct to define a position.

Reply:
I mean scan count increasing and `current` updating should be atomic. ok for now.